CHANGELOG.md

```diff
@@ -14,7 +14,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [#79](https://github.com/nf-core/atacseq/issues/79) - Parallelize DESeq2
 * [#80](https://github.com/nf-core/atacseq/pull/80) - Added social preview image
 * Update template to tools `1.9`
-* Parameter `--skip_consensus_peaks` to skip consensus peak generation
+* Parameters:
+  * `--skip_consensus_peaks` to skip consensus peak generation
+  * `--deseq2_vst` to use `vst` transformation instead of `rlog` with DESeq2
```

Comment:
Not sure how strict you are, but technically `vst` already has transformation in its name, maybe you can use 'normalization' instead. Same for other occurrences later on.

Reply:
Fixed https://github.com/nf-core/atacseq/pull/93/commits/a1516de9689401afd84e5a2617d08ba539e9765f